src/textures/Texture3D.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @author Artur Trzesiok
+ */
+
+import { Texture } from './Texture.js';
+import { NearestFilter } from '../constants.js';
+
+function Texture3D( data, width, height, length, format, type, mapping, wrapS, wrapT, magFilter, minFilter, anisotropy, encoding ) {
+
+	Texture.call( this, null, mapping, wrapS, wrapT, magFilter, minFilter, format, type, anisotropy, encoding );
+
+	this.image = { data: data, width: width, height: height, length: length };
```

Comment:
Is there any reasons why you're targeting only data texture, not image texture?

Reply:
Very simple reason - i only need a data texture. I the other hand - writing support for 3d images dataformat is a separate task from my point of view.

Reply:
OK. Then I think `DataTexture3D` would sound better rather than `Texture3D` in this PR.

Reply:
Would there be a significant difference between a DataTexture3D and just a Texture3D in this case? The width, height, and depth need to be separately specified even if using an image to bind data:
https://developer.mozilla.org/en-US/docs/Web/API/WebGL2RenderingContext/texImage3D

As a little bit of an aside, how is the data interpreted when binding a 2d image as a texture 3d?